Modules/_datetimemodule.c

```diff
@@ -1939,8 +1944,46 @@ wrap_strftime(PyObject *object, PyObject *format, PyObject *timetuple,
             ptoappend = PyBytes_AS_STRING(freplacement);
             ntoappend = PyBytes_GET_SIZE(freplacement);
         }
+#ifdef Py_NORMALIZE_CENTURY
+        else if (ch == 'Y' || ch == 'G') {
+            /* 0-pad year with century as necessary */
+            PyObject *year_str;                     /* py string, year */
+            PyObject *year;                         /* py int, year */
```

Comment:
Can you please declare these in the L1963-L1977 scope?

Reply:
Ah you're right. Gotta pay better attention to in which scope the actual usage is.